tests/test_optim/test_optimizer/test_optimizer.py

```diff
@@ -51,6 +51,9 @@ def __init__(self):
         self.sub = SubModel()
         self.conv3 = nn.Sequential(nn.Conv2d(3, 4, kernel_size=1, bias=False))
         self.conv3[0] = self.conv1[0]
+        if not duplicate_model_require_grad:
+            self.conv1[0].requires_grad_(False)
+            self.conv3[0].requires_grad_(False)
```

Comment:
I think it could be more proper to set `model.conv1[0].requires_grad_(False)` in the unit test function rather than modify the implementation of `ExampleDuplicateModel`